arch/arm64/boot/dts/qcom/msm8953-xiaomi-tissot.dts

```diff
@@ -11,11 +11,42 @@
 /delete-node/ &qseecom_mem;
 /delete-node/ &wcnss_fw_mem;
 
+/* Decrease the size */
+&cont_splash_mem {
```

Comment:
I don't know if will work in this way. You are calling the node and override its value, If I remember correctly you can only add attributes, not override them.

Reply:
There is no point in redefining it. The address remains the same. Maintainers will definitely tell you about this when trying to upstream

Reply:
Ok